code/divide_conquer/Quick_Sort/quick_sort.hs

```diff
@@ -0,0 +1,5 @@
+quicksort :: (Ord a) => [a] -> [a]
```

Comment:
Please add the comment  /* Part of Cosmos by OpenGenus Foundation */ 
at the top of the code you contribute in this organization ! 👍